File: app/src/main/java/com/stevenschoen/putionew/model/responses/AccountInfoResponse.java
package com.stevenschoen.putionew.model.responses;

import com.stevenschoen.putionew.model.ResponseOrError;
import com.stevenschoen.putionew.model.account.PutioAccountInfo;

public class AccountInfoResponse extends ResponseOrError.BasePutioResponse {
	private PutioAccountInfo info;

	public PutioAccountInfo getInfo() {
		return info;
	}
}

File: app/src/main/java/com/stevenschoen/putionew/model/responses/AccountInfoResponse.kt
package com.stevenschoen.putionew.model.responses

import com.stevenschoen.putionew.model.ResponseOrError
import com.stevenschoen.putionew.model.account.PutioAccountInfo

class AccountInfoResponse(
    val info: PutioAccountInfo
) : ResponseOrError.BasePutioResponse()
